chapters/09-package-website.Rmd

```diff
@@ -5,6 +5,77 @@
 - Getting all resources and material organized
 - Defining lifecycle of your package and individual functions (with lifecycle)
 
+## Learning Objectives
+
+* Describe the functionality `pkgdown` provides to deploy a website for package documentation
+* Customize the website with pre-existing themes
+* Organize the website with vignettes, tutorials and custom pages
+* Explain a functions' lifecycle process and the tools to handle it in R
+
+## Outline
+
+* `pkgdown`
+
+  * Text: What is `pkgdown`?
+  * Text: Benefits of having documentation be code-generated
+  * Text: Benefits of having a public website with the package docs
+  * Exercise: Build and serve the website locally
+
+* Website Customization
+
+  * Text: Why does making a package website standout is useful
+  * Text: Where do the `pkgdown` themes come from?
+  * Exercise: Change the theme of the website to one different from the default
+  * Text: Organizing beyond themes
+      * Text: Transition to talking about sections, tutorials and vignettes
+
+* Functions' Lifecycle
+  
+  * Text: Explain what is the lifecycle of a function
+  * Text: Explain the different lifecycle stages
+    * Text: {experimental --> stable --> either deprecated or superseded}
+  * Exercise: Search for a superseded function in a tidyverse package
+    * We can specify which package later
+  * Text: Walkthrough of the process of deprecating a function
+
+## Final exercise
+
+The purpose of this exercise is for the participant to go through a successful deployment of a customized `pkgdown` website and documenting their function's lifecycles.
+
+A. Setting up a website (with pkgdown)
+
+1. Set up the `pkgdown` infrastructure
+> `usethis::use_pkgdown()`
+
+2. Build the site locally
+> `pkgdown::build_site()`
+
+3. Serve the site locally
+> `pkgdown::serve_site()`
+
+B. Customizing the themes
+
+4. Change the default theme using `_pkgdown.yml`
+> Add the yaml inside the file
+
+5. Add a custom title and description for the Home Page
+> Modify `home.title` and `home.description` inside of `_pkgdown.yml`
+
+C. Getting all resources and material organized
+
+6. Organize functions in the reference section
+> Modify `_pkgdown.yml` to add custom sections to separate functions
+
+6(b). Organize tutorials and vignettes (optional)
+> Modify `_pkgdown.yml` to add custom sections to organize tutorials and vignettes.
+
+D. Defining lifecycle of your package and individual functions (with lifecycle)
+
+7. Add "experimental" lifecycle tag to one of the developed functions
+> Modify the functions Roxygen annotations; `@description \\`r lifecycle::badge("experimental")\\``
+
+8. Commit the changes and push up to GitHub. Confirm that the package website has been built.
+
```

Comment:
Is this a replacement for the Final exercise (below) or in addition to?  It looks really good as a standalone replacement for the old final exercise to me!

Reply:
It is a "reviewed" version of the original final exercise. I'll do a cleanup in the next commit.